src/pages/studyView/charts/barChart/BarChart.tsx

```diff
@@ -256,6 +257,31 @@ export default class BarChart extends React.Component<IBarChartProps, {}>
     }
 
     public render() {
+        const containsNA = (element: BarDatum): boolean => {
+            return element.dataBin.specialValue === 'NA';
+        };
+        const doesNotContainNA = (element: BarDatum): boolean => {
+            return !containsNA(element);
+        };
+        const collectCount = (
```

Comment:
looks to me like what you are really doing here is map and then join(", ")?

Reply:
that's correct; I don't know if NA is guaranteed to show up once in this.barData, so I added logic to display the values of all instances of NA entries